### ISSUE TITLE
Ensure keyboard focus is visible in all functional items which receive keyboard focus

### ISSUE THREAD

**Issue by Joetar** [Beep6581/RawTherapee#6866]
RawTherapee version 5.9
Windows 11 Home OS, version 22H2 22621.2428

Currently it seems that functional elements receive keyboard focus as they should but the keyboard focus is not visible in all elements. For example, the tools that are located above the image being worked, do not have a visible focus. 
![image](https://github.com/Beep6581/RawTherapee/assets/60326078/e2b66b8e-b40c-4805-8ab2-7ec4e5392c1c)
If the user randomly presses Enter whilst tabbing through the UI, those tools become activated so they receive keyboard focus, but just don't show it.

The problem can be reproduced simply by opening the program and then starting to move around with tab key.
In order to test that particular tool row, upload a raw image first by using a mouse (easiest) and then start tabbing and occasionally press Enter. (Cannot specify how many times tab must be pressed in order to reach those tools as focus visibility is so poor it is impossible to tell where it is going.)

Keyboard focus should be visible at all functional elements. And to ensure accessibility and usability, the contrast ratio between the focus indicator and its adjacent colours should be more than 3:1 and the focus indicator should be at least 2 CSS pixels thick.

Suggested solution:

Add a style for focused elements in your CSS styles. For theme "RawTherapee" e.g.:
:focus {
     border: 2px solid rgb(196, 196, 196); 
}

This solution would fulfill the accessibility requirements upheld by W3C [](https://www.w3.org/TR/WCAG22/#focus-appearance)
